docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Polling Tentacles over Standard HTTPS Port
```

Comment:
Hi Mark, I thought that I had just changed the title on the page. So, I am a bit puzzled. I Bing chat search is talking about editing the yaml in the config file. Not confident I want to do this and quite possibly make matters worse. So any pointers would be greatly appreciated. Thanks.

Reply:
No worries @JonPHardy57 - I've fixed up the changes

The tl;dr of what happened is that when you created the branch initially, this was done from the `master` branch. Since the master branch has been around, the docs have now moved to the `main` branch, which includes a slightly different page structure.

When you made your change and then merged main into your `Update-Title` branch - you got both pages, your "renamed" title one, and the existing one (that had moved location in the filesystem). You probably had some merge conflicts to resolve with the initial master -> `Update-Title` branch and kept your file with the new name as after all that's what you wanted to change!

So all I did, was delete your "new" page and apply the title change to the existing page.

Moving forward there its very unlikely this could happen again as @steve-fenton-octopus has deleted the `master` branch just now - I'd recommend you delete the master branch from your remote fork/local git repo too!

Hope that helps explain! :smile: